boards/arm/nrf9160_pca20035/nrf9160_pca20035_defconfig

```diff
@@ -23,7 +23,6 @@ CONFIG_UART_CONSOLE=y
 
 # Enable I2C
 CONFIG_I2C=y
-CONFIG_I2C_2=y
```

Comment:
@anangl : Does this look good to you?

Reply:
My limited understanding is that this should be okay due to this one enabling `I2C_2` if `I2C` is enabled (or not disabled by MCUboot):
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/boards/arm/nrf9160_pca20035/Kconfig.defconfig#L21

Reply:
> @anangl : Does this look good to you?

With the above explanation, it makes sense. Maybe then some comment in this file explaining why `CONFIG_I2C_2=y` is no longer there, could be helpful for the future?

Reply:
> > @anangl : Does this look good to you?
> 
> With the above explanation, it makes sense. Maybe then some comment in this file explaining why `CONFIG_I2C_2=y` is no longer there, could be helpful for the future?

Yep, that's reasonable, I've added a comment about it.